src/de.rs

```diff
@@ -1222,39 +1272,70 @@ impl<'de, 'document> de::Deserializer<'de> for &mut DeserializerFromEvents<'de,
             }
             parse_tag(tag)
         }
+        // HACK: switch to JSON enum semantics for JSON content
+        // Robust impl blocked on https://github.com/serde-rs/serde/pull/2420
```

Comment:
Sorry, this is a personal preference, but this thing is undiscoverable, could you use `// TODO: ...`.

I sometimes grep for TODO if I want to work on a project and am out of tasks. I will never grep for HACK.

Thanks.